src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetFiltersMappingPage.tsx

```diff
@@ -31,14 +32,26 @@ import PageNav, { NavItem } from '@common/components/PageNav';
 import { Dictionary } from '@common/types';
 import useDebouncedCallback from '@common/hooks/useDebouncedCallback';
 import { useQuery } from '@tanstack/react-query';
+import camelCase from 'lodash/camelCase';
 import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import { generatePath, useParams } from 'react-router-dom';
 import omit from 'lodash/omit';
 import { useImmer } from 'use-immer';
 import sumBy from 'lodash/sumBy';
-import kebabCase from 'lodash/kebabCase';
 import compact from 'lodash/compact';
 
+export const sectionIds = {
```

Comment:
Seems like a neat way of keeping the nav and the links consistent.  Do we do this pattern elsewhere at all?